phantom-jdk8/src/main/scala/com/outworkers/phantom/jdk8/package.scala

```diff
@@ -56,9 +65,20 @@ package object jdk8 {
   )(Primitives.LocalDateIsPrimitive)(CQLSyntax.Types.Date)
 
   implicit val zonedDateTimePrimitive: Primitive[ZonedDateTime] = {
-    Primitive.derive[ZonedDateTime, (Long, String)](dt => dt.toInstant.toEpochMilli -> dt.getZone.getId) {
-      case (timestamp, zone) => ZonedDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneId.of(zone))
-    }
+    val tuplePremitive = implicitly[Primitive[(Long, String)]]
+    Primitive.manuallyDerive[ZonedDateTime, (Long, String)](
+      dt => dt.toInstant.toEpochMilli -> dt.getZone.getId,
+      {
+        case (timestamp, zone) => ZonedDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneId.of(zone))
+      }
+    )(tuplePremitive)(
+      CQLSyntax.Collections.tuple +
```

Comment:
I think we might be able to use the `QueryBuilder` here instead of adding inline string logic. Other than that, looks fine.